lib/CL/pocl_llvm_build.cc

```diff
@@ -1045,8 +1045,13 @@ int pocl_invoke_clang(cl_device_id Device, const char** Args) {
 
   const char **ArgsEnd = Args;
   while (*ArgsEnd++ != nullptr) {}
+  llvm::SmallVector<const char*, 0> ArgsArray(Args, ArgsEnd);
 
-  llvm::ArrayRef<const char*> ArgsArray(Args, ArgsEnd);
+  std::string ld_path;
```

Comment:
Let's converge towards the C++ style guide here, so please use CamelCase for variable names in C++ files.
```suggestion
  std::string LDPath;
```